classes/CreateMailLog.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace SureSoftware\MailLog\Classes;
```

Comment:
Given that these are just helpers for the MailLog Model, I would personally put these methods on the MailLog Model instead

Reply:
I have moved it to the model.